### DISCUSSION TITLE
Improve itowns examples structuration

### DISCUSSION THREAD

**Discussion by jailln** [iTowns/itowns#2070] [Ideas]
There is currently a lot of [examples in itowns](https://www.itowns-project.org/itowns/examples/), some are more functional and other are more technical. In addition, examples' descriptions are limited to their name. Also, on many examples, there is no explanation about what interaction the user should do and what he should see. For all these reasons, I think it's quite hard for new users (especially non technical or non specialist ones) to use the examples and to see what itowns is capable of. Some iTowns examples are also kind of tutorials (e.g. [custom controls](https://github.com/iTowns/itowns/blob/master/examples/misc_custom_controls.html)), and their main purpose is to show how to implement something specific with itowns.

Therefore, I think we should improve the way we show what itowns allows to do.

I see two needs:
1. The first one is meant for people that don't know itowns and just want to discover what it can do quickly. The purpose is to attract them to make them want to use itowns. To do so I think we should quickly show all the potential of itowns and guide them. For this need I think we should make one example (or two or three max) showing the main possibilities of itowns in the form of a guided tour, moving the user around in a 3D scene, showing different datasets (nice ones, with good textures, scaled up, etc.) and that explain the possible interactions with info text above the scene, etc. This example should probably be showcased in the home page of the website.
2. The second one is meant for users. The point here is to explain to them how to use itowns. For this one I think we should only write tutorials (and transform the current examples to tutorials). However, I think we should modify a bit how the tutorials are currently structured to have something more dynamic: e.g. something more like a jupyter notebook or like [mapbox examples](https://docs.mapbox.com/mapbox-gl-js/example/navigation/) but with more text explaining the code or like [Cesium Sandcastle](https://sandcastle.cesium.com/?src=3D%20Models.html) but also with more text explanation.

@itowns users and contributors, what's your opinion on this?

**Comment by valentinMachado** [comment]
Sounds good and sounds like [another website](https://threejs.org/), I would just say that these showcase projects should not belong to the itowns github repository but should be project aside.

For a no developer it would made no differences since he is going to browse itowns website in any cases, for developers it would keep the itowns folder dry and clean for their usage.

**Comment by mgermerie** [comment]
Sounds good to me as well ! :+1:
